src/main/kotlin/com/example/cms/domains/preparedcontents/services/beans/PreparedContentServiceBean.kt

```diff
@@ -66,7 +66,7 @@ class PreparedContentServiceBean @Autowired constructor(
         this.validate(entity)
         val placeholders = entity.placeholderValues.mapValues {
             if (it.key == DYNAMIC_CONTENT_KEY) {
-                HtmlTable.fromJson(it.value, listOf("dynamic-table"))
+                HtmlTable.fromJson(it.value, entity.cssStyleClasses?.split(" ") ?: listOf("dynamic-table"))
```

Comment:
` HtmlTable.fromJson(it.value, entity.cssStyleClasses?.split(" ") `

To be honest, we could just take a string as input in library, there's no need to take list. but for now it looks algiht.

Reply:
@sayemkcn vai to be honest same thing me also thinking